src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/LocationScheduleTest.kt

```diff
@@ -101,44 +101,6 @@ class LocationScheduleTest {
     }.message isEqualTo "The end time must come after the start time."
   }
 
-  @Test
-  fun `should fail to amend if causes a duplicate row`() {
```

Comment:
This test was floored anyway (regardless of the bug).

Unfortunately we cannot unit test as it relies on ID checking.  We cannot set or change the IDs to be able to test this.

Reply:
We lack integration tests in this area of the service.